<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Username</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_username</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_username"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_pass</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pass">
           <property name="maxLength">
            <number>4</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
